tests/Policy_1.py

```diff
@@ -22,11 +22,15 @@
 #
 
 import unittest
+import os.path
 
 import lsst.utils.tests
 from lsst.pex.policy import Policy, NameNotFound
 import lsst.pex.exceptions
 
+
+proddir = lsst.utils.getPackageDir('pex_policy')
```

Comment:
I think our standard calls for CamelCase for module-level variables. I'd also be happier with a more expressive name, e.g. `PexPolicyDir`.